textx/model.py

```diff
@@ -509,5 +555,15 @@ def call_obj_processors(model_obj):
         if parser.debug:
             parser.dprint("CALLING OBJECT PROCESSORS")
         call_obj_processors(model)
+    
+    if type(model) not in PRIMITIVE_PYTHON_TYPES:
+        # Cross-references for go-to definition language server support
+        # Already sorted based on ref_pos_start attr (required for binary search)
+        model._pos_crossref_list = pos_crossref_list
```

Comment:
I think we should control this mechanism using parameter during meta-model creation. The default would be to turn it off. The rationale is that this might consume a lot of memory for larger models while it will be usable just for textX tools at the moment.